ci-operator/config/opendatahub-io/text-generation-inference/opendatahub-io-text-generation-inference-main.yaml

```diff
@@ -0,0 +1,38 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- build_args:
+  - name: DOCKER_BUILDKIT
+    value: "1"
+  - name: BUILDKIT_PROGRESS
+    value: plain
+  - name: PROTOC_VERSION
+    value: "23.2"
+  context_dir: .
+  dockerfile_path: Dockerfile
+  from: src
+  to: text-generation-inference
+promotion:
+  namespace: opendatahub-io
+  tag: latest
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: fast
+      version: "4.12"
+resources:
+  '*':
+    limits:
+      cpu: "20"
+      memory: 16Gi
+    requests:
+      cpu: "20"
+      memory: 16Gi
```

Comment:
After you build the tgis image, you should send it to opendatahub quay.io
in order to send the built image, please refer this https://github.com/openshift/release/blob/master/ci-operator/config/opendatahub-io/modelmesh-serving/opendatahub-io-modelmesh-serving-main.yaml#L78-L95

with this, we can see pr tag/fast tag

Reply:
done